module2/homework/insensitivePalindrom/palindrome.cpp

```diff
@@ -0,0 +1,16 @@
+#include "palindrome.hpp"
+
+#include <algorithm>
+#include <cctype>
```

Comment:
You should include string here according to "include-what-you-use" rule.

Reply:
I don't see any sense to do that. It is included in hpp and including it here again doesn't give any benefit and doesn't make code more readable. I think that Mateusz said not to double what is included in hpp file.